File: day_four/happyBirthday.cpp
#include <iostream>

int main() {

    std::string name;

    std::cout << "Happy birthday! What is your name?: \n";
    std::getline(std::cin, name);

    std::cout << "Happy birthday to you!\n";
    std::cout << "Happy birthday to you!\n";
    std::cout << "Happy birthday dear " << name << "!\n";
    std::cout << "Happy birthday to you!\n";


    return 0;
}

File: day_four/happy_birthday.py
#!/usr/bin/python3

import sys

def happy_birthday(name):
    print("Happy birthday to you!")
    print("Happy birthday to you!")
    print(f"Happy birthday dear {name}!")
    print("Happy birthday to you!")

def main():
    name = sys.argv[1]
    happy_birthday(name)


if __name__=="__main__":
    main()